### PR TITLE
Connect nosql cosmos db

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**
Connected azure/cosmosDb and implemented CRUD for task entit
pm tests jpg
y

### CODE REVIEW

File: backend/Tasque.BLL/Interfaces/ITaskService.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
```
Comment (Its-OP) [inline backend/Tasque.BLL/Interfaces/ITaskService.cs:1]: We have an ICrudEntityService (or something like this) which is done to not create other standard CRUD services

**Review by Its-OP — CHANGES_REQUESTED** [review]
Can we use Int ids in Cosmos DB ? It would make further support much easier, as it'll be possible to reuse our existing interfaces.

File: backend/Tasque.BLL/Interfaces/ITaskService.cs (hunk 1 above)
Comment (DaniilPoiarkov) [reply backend/Tasque.BLL/Interfaces/ITaskService.cs:1]: I decided to make another one with full implementration coz for this entity we use two data bases and there are some bottlenecks to make it work as expected

File: backend/Tasque.BLL/Services/TaskService.cs [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            {
+                CustomDateFields = attributes.CustomDateFields,
+                CustomTextFields = attributes.CustomTextFields,
+                CustomParagraphFilds = attributes.CustomParagraphFilds,
+                CustomNumberFields = attributes.CustomNumberFields,
+
+                CustomCheckboxFields = attributes.CustomCheckboxFields,
+                CustomDropdownFields = attributes.CustomDropdownFields,
+                CustomDropdownDependenciesFields = attributes.CustomDropdownDependenciesFields,
+            };
```
Comment (Anatolii-Ivanov) [inline backend/Tasque.BLL/Services/TaskService.cs]: Can we make array this time?...

File: backend/Tasque.Core.WebAPI/Program.cs [hunk 3]
```diff
@@ -9,7 +9,7 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-builder.Logging.ClearProviders();
+//builder.Logging.ClearProviders();
```
Comment (Anatolii-Ivanov) [inline backend/Tasque.Core.WebAPI/Program.cs]: Remove this line

**Comment by DaniilPoiarkov** [comment]
> Can we use Int ids in Cosmos DB ? It would make further support much easier, as it'll be possible to reuse our existing interfaces.

No, only string is available

**Review by Its-OP — APPROVED** [review]

**Pull request closed without merging**